code/datums/wires/syndicatebomb.dm

```diff
@@ -42,9 +46,12 @@
 				playsound(B, 'sound/machines/click.ogg', 30, 1)
 				B.active = TRUE
 				B.update_icon()
+			if(B.delayedlittle)
+				holder.visible_message("<span class='notice'>\icon[B] Nothing happens.</span>")
 			else
 				holder.visible_message("<span class='notice'>\icon[B] The bomb seems to hesitate for a moment.</span>")
-				B.timer += 5
+				B.timer += 10
+				B.delayedlittle = TRUE

```

Comment:
Thanks, fixed now. It's pretty rare situation so I hadn't even thought about it.
Should I add a similar check to the delay wire so bigdelay cannot be activated before even arming the bomb or shall we use it like this?